Fabric.Authorization.Domain/Resolvers/Permissions/RolePermissionResolverService.cs

```diff
@@ -78,6 +78,16 @@ public RolePermissionResolverService(RoleService roleService)
             };
         }
 
+        private static bool BypassRoleEvaluation(Role role, List<string> groupNames, PermissionResolutionRequest permissionResolutionRequest)
+        {
```

Comment:
This was missing originally, but should `|| role.Permissions.Count == 0` also be in this condition?

Reply:
That looks like a good performance optimization. I'll add it in.